### ISSUE TITLE
Types suffixed with name 'EventArgs' should inherit from 'EventArgs'

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#639]
We should report in case a type has a name that ends with `EventArgs` but does not inherit from `System.EventArgs`.

By indicating that the type is an event argument, developers are fooled in case the type is not.

Following code should trigger an issue:
```c#
public class FakeEventArgs
{}
```

While following code should **NOT** trigger an issue because it inherits from `EventArgs`:
```c#
public class MyEventArgs : EventArgs
{}
```
